charts/rancher-monitoring/103.0.0+up40.1.2/charts/hardenedNodeExporter/Chart.yaml

```diff
@@ -11,4 +11,4 @@ description: Sets up a deployment of the PushProx proxy and a DaemonSet of PushP
   clients.
 name: hardenedNodeExporter
 type: application
-version: 0.1.6
+version: 102.0.2
```

Comment:
If you're going to change the version to 103.0.0, this version also needs to be updated.